src/cdk/a11y/BUILD.bazel

```diff
@@ -0,0 +1,14 @@
+package(default_visibility=["//visibility:public"])
+load("@angular//:index.bzl", "ng_module")
+
+ng_module(
+  name = "a11y",
+  srcs = glob(["**/*.ts"], exclude=["**/*.spec.ts"]),
+  module_name = "@angular/cdk/a11y",
+  deps = [
+    "//src/cdk/coercion",
+    "//src/cdk/keycodes",
+    "//src/cdk/platform",
+  ],
+  tsconfig = ":tsconfig-build.json",
```

Comment:
note, you might want to try using the same tsconfig for all the rules, and have it be the one the editor uses too

Reply:
Impossible presently because we need to specify a different `flatModuleId` for each package, but would be nice long term.